tools/run-mypy

```diff
@@ -16,6 +16,7 @@ TOOLS_DIR = os.path.dirname(os.path.abspath(__file__))
 os.chdir(os.path.dirname(TOOLS_DIR))
 
 exclude_common = """
+zerver/tests/webhooks/test_mailchimp.py
```

Comment:
It's being worked on by Zulip Devs so I was trying to just test the documentation since that's what i am working on right now.